### DISCUSSION TITLE
NDS/Drastic problems SOLVED

### DISCUSSION THREAD

**Discussion by dariu76** [OnionUI/Onion#1657] [Support]
Hello everyone!
after switching from onion to 4.3 and 4.3.1-1, pressing select+start no longer opens the drastic menu for me, and with menu+start no longer opens the custom menu.
the other commands work correctly, but without access to these two menus i am very limited! has this happened to anyone else? do you have any solutions?
in desperation, i even did a clean reinstall of everything, but the menus are still not accessible. 
i have a miyoo mini plus, version 202306282128, mode MY354.
I also have a bug with the volume: pressing volume up brings the volume to maximum (green bar colour), pressing volume down the bar turns red and the sound goes off, it doesn't go down. this doesn't happen all the time and with all emulators, sometimes it works normally, but it happens often.
thanks to anyone who can help me!

**Comment by thomasklemenc** [comment]
Hey,

the NDS hotkeys are:

DraStic menu: Menu + Select
Custom menu: Menu + Start
https://onionui.github.io/docs/emulators/nds

both work fine here. Not sure aboutthe volume bar, don't think I've seen that one yet
